extensions/integration-tests/src/testConfig.ts

```diff
@@ -93,6 +103,16 @@ export async function getDefaultTestingServer(): Promise<TestServerProfile> {
 	return servers[0];
 }
 
+export async function getAzureServer(): Promise<TestServerProfile> {
+	let servers = await getTestingServers();
+	return servers.filter(s => s.version === '2012')[0];
+}
+
+export async function getStandaloneServer(): Promise<TestServerProfile> {
+	let servers = await getTestingServers();
+	return servers.filter(s => s.version === '2017')[0];
```

Comment:
What's the purpose of these filters? This doesn't seem very robust long term - what if we want to test against a different version?